search.go

```diff
@@ -163,28 +163,12 @@ func getPackageOutput(packagesList map[string]map[string]util.Package) ([]byte,
 	}
 	sort.Strings(keys)
 
-	var output []map[string]interface{}
+	var output []util.BasePackage
 
 	for _, k := range keys {
 		parts := strings.Split(k, separator)
 		m := packagesList[parts[0]][parts[1]]
```

Comment:
nit: is there is a need to check the number of parts here (in case if it's empty)?

Reply:
I also stumbled over this line when I did the change and I plan to follow up to make this nicer. There must be a better way here.